specs/www.rst

```diff
@@ -223,12 +246,16 @@ Connection
 Sent when the client initially opens a connection and is about to finish the
 WebSocket handshake.
 
-This message must be responded to with either an *Accept* message
-or a *Close* message before the socket will pass ``websocket.receive``
-messages. The protocol server must send this message
-during the handshake phase of the WebSocket and not complete the handshake
-until it gets a reply, returning HTTP status code ``403`` if the connection is
-denied.
+This message must be responded to with either an *Accept* message, a
+*Response Start* message, or a *Close* message. If a Response Start
+message is sent it should be followed by *Response Body* messages as
+required by the HTTP spec. Websocket messages cannot be sent after a
+*Response Start* message. If an *Accept* message is sent the handshake
+is completed and the socket will pass ``websocket.receive``
+messages. The protocol server must send an accept message during the
+handshake phase of the WebSocket and not complete the handshake until
+it gets a reply, returning HTTP status code ``403`` if the connection
+is denied.
```

Comment:
I guess it's fine to have the websocket and http events intermingle, they're in the same document for a reason. Could you specifically say "HTTP Response Start" etc. here to make that clear?

Reply:
I've done so.